README.md

```diff
@@ -107,6 +107,7 @@
 - [Photoview](https://photoview.github.io/) - Photo gallery for self-hosted personal servers with Facial Recognition.
 - [Photostructure](https://photostructure.com/) - Self-hosted photo library that makes browsing and sharing a lifetime of memories delightful.
 - [LibrePhotos](https://github.com/LibrePhotos/librephotos) - Active [OwnPhotos](https://github.com/hooram/ownphotos) fork. Self hosted alternative to Google Photos.
+- [ownCloud](https://owncloud.com/) - ownCloud is an open-source file sync, share and content collaboration software that lets teams work on data easily from anywhere, on any device. 
```

Comment:
Can you link to the Photos plugin/app if they have one here?

Reply:
OC is in the process of recreating the frontend, so atm there is no photo app. I'll add it once there is one